README.md

```diff
@@ -1,10 +1,5 @@
-Coderwall SVG [![Donate on Gittip](http://badgr.co/gittip/twolfson.png)](https://www.gittip.com/twolfson/)
+Coderwall SVG [![Donate on Gittip](https://img.shields.io/gratipay/twolfson.svg)](https://gratipay.com/~twolfson/)
 =============
-In this repo is a quick (and sloppy) conversion of the [Coderwall](http://coderwall.com/) logo to SVG
+In this repo is a quick conversion of the [Coderwall](http://coderwall.com/) logo to SVG
 
-![Coderwall logo](https://raw.github.com/twolfson/coderwall-svg/master/coderwall.png)
-
-Download Links
---------------
-
-[svg](https://raw.github.com/twolfson/coderwall-svg/master/coderwall.svg) [png](https://raw.github.com/twolfson/coderwall-svg/master/coderwall.png)
```

Comment:
Please move back to the original split format of logo and download links. It was much easier for someone to get a link to download the resource (e.g. don't need to click to the page then click "raw" then click download).